Dockerfile

```diff
@@ -96,6 +96,7 @@ RUN mkdir /usr/local/lib/swan && \
 EXPOSE 8888
 
 ENV SHELL /bin/bash
+ENV SWAN_IMAGE_VERSION 3.2
```

Comment:
Can you remind me what was the use of this? Since this will require maintenance every time we produce a new image version.

Reply:
I.e. in the future to save that information inside project.

Reply:
Can we leave this out for the moment until we actually need it?

Reply:
done